File: bee1044.js
const input = require("fs").readFileSync("stdin", "utf-8")
const lines = input.split(" ")

const A = Number(lines.shift())
const B = Number(lines.shift())

if(A % B == 0 || B % A == 0) {
    console.log("Sao Multiplos")
} else {
    console.log("Nao sao Multiplos")
}

File: python/bee1044.py
# -*- coding: utf-8 -*-

m = [int(x) for x in input().split(" ")]

if m[0] % m[1] == 0 or m[1] % m[0] == 0:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")